xgcm/grid.py

```diff
@@ -1024,6 +1038,11 @@ def interp(self, da, axis, **kwargs):
         ----------
         axis : str
             Name of the axis on which to act
+        conserve : str or tuple of str
+             Axis along to conserve data. The data will be multiplied with the metric
+             appropriate to `conserve` and the interpolated data will be divided by
+             the same metric, except located at the new grid location
+             (the default is False).
```

Comment:
Let's eliminate discussion of conservation and go with a simpler name here.
```
metric_weighted : str or tuple of str
    If an axis or list of axes is specified, the grid metrics will be used to determined the weight for interpolation.
    If `False` (default), the points will be weighted equally.
```

Reply:
Good suggestion. Done